src/collection-view.js

```diff
@@ -603,39 +556,77 @@ const CollectionView = Backbone.View.extend({
     });
   },
 
-  // Renders each view in children and creates a fragment buffer from them
-  _getBuffer(views) {
-    const elBuffer = this.Dom.createBuffer();
-
-    _.each(views, view => {
-      renderView(view);
-      this.Dom.appendContents(elBuffer, view.el, {_$contents: view.$el});
-    });
-
-    return elBuffer;
-  },
-
   // Override this method to do something other than `.append`.
   // You can attach any HTML at this point including the els.
   attachHtml(els) {
     this.Dom.appendContents(this.$container[0], els, {_$el: this.$container});
   },
 
+  isEmpty() {
```

Comment:
here's the major change to the empty view.. the rest of the methods below were just moved.